### ISSUE TITLE
Add location_type to API

### ISSUE THREAD

**Issue by aaronantrim** [transitland/transitland-datastore#246]
I am not sure if your team is yet interested in public feedback, but I noticed that location_type is not returned by the API in the list of stops:
http://transit.land/api/v1/stops?servedBy=o-9q9-bart

It would be useful to see whether a location is a stop, station, or station entrance, and further, to be able to filter by location_type, and by parent station.

**Comment by drewda** [comment]
@aaronantrim public feedback is quite welcome (although we may not be able to act on every request).

Coming soon on our development roadmap is a station hierarchy that will model platforms, egress points, and other aspects of complex stations. More information about the preliminary data model is at: https://github.com/transitland/transitland-datastore/blob/station-hierarchies/doc/station-hierarchies.md

In short, this new data model will allow for the type of querying that you're suggesting.

**Comment by drewda** [comment]
Will be implemented by #256
